apps/src/templates/levelSummary/FreeResponseAiSummaryBox.tsx

```diff
@@ -72,8 +72,8 @@ const FreeResponseAiSummaryBox: React.FC<FreeResponseAiSummaryBoxProps> = ({
       <BodyTwoText>
         <strong>{`${i18n.reasoning()}:`}</strong>
         {proficiencyCount > proficienceyThreshold
-          ? 'This is proficient because more than 80% of the students demonstrated proficiency in their responses. '
-          : 'This is needs review less than 80% of students demonstrated proficiency in their responses. '}
+          ? 'More than 80% of the students demonstrated proficiency in their responses. '
+          : 'Less than 80% of the students demonstrated proficiency in their responses. '}
```

Comment:
We could get fancy and make the % based on ` proficienceyThreshold` which makes this ever so _slightly_ easier to maintain, but I don't feel strongly about it and we're going to have to touch this again for i18n if we scale.